TelCo.ColorCoder/Test.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+using TelCo.ColorCoder;
+
+namespace TelCo.ColorCoder
+{
+    /// <summary>
+    /// This class provides the testinf and debugging
+    /// </summary>
+    partial class Program
+    {
+        /// <summary>
+        /// Test code for the class
+        /// </summary>
+        /// <param name="args"></param>
+        private static void Main(string[] args)
+        {
+            int pairNumber = 4;
+            ColorPair testPair1 = Program.GetColorFromPairNumber(pairNumber);
+            Console.WriteLine("[In]Pair Number: {0},[Out] Colors: {1}\n", pairNumber, testPair1);
+            Debug.Assert(testPair1.majorColor == Color.White);
+            Debug.Assert(testPair1.minorColor == Color.Brown);
+
+            pairNumber = 5;
+            testPair1 = Program.GetColorFromPairNumber(pairNumber);
+            Console.WriteLine("[In]Pair Number: {0},[Out] Colors: {1}\n", pairNumber, testPair1);
+            Debug.Assert(testPair1.majorColor == Color.White);
+            Debug.Assert(testPair1.minorColor == Color.SlateGray);
+
+            pairNumber = 23;
+            testPair1 = Program.GetColorFromPairNumber(pairNumber);
+            Console.WriteLine("[In]Pair Number: {0},[Out] Colors: {1}\n", pairNumber, testPair1);
+            Debug.Assert(testPair1.majorColor == Color.Violet);
+            Debug.Assert(testPair1.minorColor == Color.Green);
+
+            ColorPair testPair2 = new ColorPair() { majorColor = Color.Yellow, minorColor = Color.Green };
+            pairNumber = Program.GetPairNumberFromColor(testPair2);
+            Console.WriteLine("[In]Colors: {0}, [Out] PairNumber: {1}\n", testPair2, pairNumber);
+            Debug.Assert(pairNumber == 18);
+
+            testPair2 = new ColorPair() { majorColor = Color.Red, minorColor = Color.Blue };
+            pairNumber = Program.GetPairNumberFromColor(testPair2);
+            Console.WriteLine("[In]Colors: {0}, [Out] PairNumber: {1}\n\n", testPair2, pairNumber);
+            Debug.Assert(pairNumber == 6);
+            
+            Program varToAccessNonStatic = new Program();
```

Comment:
Naming convention - name the variable to indicate what it is.

Program program = new Program(); is more accepted nameing convention